src/src.pro

```diff
@@ -31,9 +31,11 @@ macx: {
     include(../3rdparty/luazip/src.pri)
 }
 
+include("../3rdparty/edbee-lib/edbee-lib/edbee-lib.pri");
+
 # Set the current Mudlet Version, unfortunately the Qt documentation suggests
 # that only a #.#.# form without any other alphanumberic suffixes is required:
-VERSION = 3.1.0
+VERSION = 3.0.1
```

Comment:
Also - `TScriptEditorManager.cpp` and `TScriptEditorManager.h` are missing from the relevant file listings further down this file - not clear how the project builds without them being picked-up by `qmake` when it processes this file to make the `Makefile`! 😕 

Haven't checked the CMake equivalent file(s).